src/pages/Admin/Venue/Details/VenueDetails.tsx

```diff
@@ -154,6 +154,7 @@ const VenueDetails: React.FC<VenueDetailsProps> = ({ venue }) => {
 
   const handleEditRoomSave = async (values: RoomData_v2, index: number) => {
     const newData = {
+      ...editingRoom,
       ...values,
       x_percent: editingRoom?.x_percent,
       y_percent: editingRoom?.y_percent,
```

Comment:
If we're spreading `...editingRoom` here now, is there any reason to keep the explicit `x_percent`/etc keys below?

Should this be happening before `...values`, or after it?

Reply:
We could avoid this in a new way. Just to update the only properties from edit form

Reply:
> Just to update the only properties from edit form

Not sure what you mean by this, can you please elaborate?

Reply:
We are spreading the `editingRoom` and are merging the `values` into one obj `newData`. It should work smoothly

Reply:
Cool cool.

Can `values` ever container `x_percent`/etc in a way that we don't want to keep it/allow it to be overwritten from what was already in `editingRoom`? That's about the only potential edgecase I can see with this change.

Reply:
`values` appears to be of type `RoomData_v2`, so at least in theory could have these keys set.